packages/neso/package.py

```diff
@@ -93,4 +95,8 @@ def cmake_args(self):
 
                     break
 
+        if "+nvcxx" in self.spec:
```

Comment:
Does this only work with hipsycl or can it be used with oneAPI as well? If it can't, you should add a [conflicts](https://spack.readthedocs.io/en/latest/packaging_guide.html#conflicts-and-requirements) statement.

Reply:
Added a conflict statement (see message for commit e2b6e79), also opened an issue if anyone wants to test the Intel OneAPI+Nvidia combination.